cmake/GetZLIB.cmake

```diff
@@ -0,0 +1,27 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+function(get_zlib SOURCE_DIR BINARY_DIR)
+    if(MINIFI_ZLIB_SOURCE STREQUAL "CONAN")
+        message("Using Conan Packager to manage installing prebuilt zlib external lib")
+        find_package(ZLIB REQUIRED)
+    elseif(MINIFI_ZLIB_SOURCE STREQUAL "BUILD")
+        message("Using CMAKE's ExternalProject_Add to manage source building zlib external lib")
```

Comment:
```suggestion
        message("Using CMake to build zlib from source")
```

Reply:
Updated this one based on your feedback.